bigquery/google/cloud/bigquery/client.py

```diff
@@ -83,6 +83,17 @@ def __init__(self, project=None, credentials=None, _http=None):
             project=project, credentials=credentials, _http=_http)
         self._connection = Connection(self)
 
+    def _clone(self, project):
+        """Create a new client for another project.
+
+        Helper for creating dataset / table instances in remote projects.
+
+        :rtype: :class:`Client`
+        :returns: a new instance, bound to the supplied project, using
+                  the same credentials / http object as this instance.
+        """
+        return self.__class__(project, self._credentials, self._http)
```

Comment:
1. Please use keyword arguments for all keywords (recall the issue we had when a GAPIC surface changed)
1. `self._http` is probably not what you want, rather `self._http_internal`